### ISSUE TITLE
The new repo has a bug.

### ISSUE THREAD

**Issue by SpongeBab** [ultralytics/yolov5#3569]
@glenn-jocher hi,I think this modify will make mistakes.

## 🐛 Bug
```
Traceback (most recent call last):
  File "E:\yolov5\train.py", line 623, in <module>
    hyp[k] = max(hyp[k], v[1])  # lower limit
KeyError: 'anchors'
```
I think it is because the following is commented in hyp.scratch,yaml.
```
 # anchors: 3  # anchors per output layer (0 to ignore)
```

## Input 
I used the pretrained weights:https://github.com/google/automl/tree/master/efficientnetv2.

I used the anchors,and it worked. 

## Output:
```
Image sizes 704 train, 704 test
Using 0 dataloader workers
Logging results to runs\train\evolve
Starting training for 300 epochs...

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
     0/299     2.57G     0.121    0.2745         0    0.3955        77       704: 100%|██████████| 350/350 [01:55<00:00,  3.04it/s]

```

**Comment by glenn-jocher** [comment]
@SpongeBab yes that is correct. If the anchor field is enabled in your hyp file then it will supercede the anchors in your model.pt or model.yaml file, so this line should mainly be uncommented when run evolution.

**Comment by glenn-jocher** [comment]
@SpongeBab good news 😃! Your original issue may now be fixed ✅ in PR #3887. This PR adds `hyp['anchors']=3` if the anchors key is missing/commented in your hyperparameter file, fixing a long-running problem when starting hyp evolution using the default command:
```
python train.py --evolve
```

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
